aws/resource_aws_iam_access_key.go

```diff
@@ -47,6 +49,40 @@ func resourceAwsIamAccessKey() *schema.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+			"ses_smtp_passwords": {
```

Comment:
Since this functionality is not part of the API, its a gray area that the Terraform AWS Provider resource supports it. Our preference in this case would be to leave it up to operators or Terraform Module designers to choose which version password is appropriate for their use case since his functionality can easily become out of date (when new regions are added) and incorrect (when regions update acceptable signatures). SES implementations outside AWS (or potentially older SES implementations in AWS partitions outside AWS Commercial) may still want/use the V2 signature format.

Let's create a simple `ses_smtp_password_v4` attribute to store the new value and remove the maintenance burden from this project. 👍 

```go
			"ses_smtp_password_v4": {
				Type:      schema.TypeString,
				Computed:  true,
				Sensitive: true,
			},
```